packages/api/public/spec/schemas/language.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  type:
+    type: string
+    enum: [language]
+  id:
+    type: string
+    example: es
+    description: The [IETF language code](https://en.wikipedia.org/wiki/IETF_language_tag).
```

Comment:
I'm thinking it would be good to follow something like [JSON:API](https://jsonapi.org/) for our API design since that will make things consistent, discoverable, and take a lot of the decision making out of the design process.

That's what I've mocked up here. We can definitely revist this if you think this format adds too much unnecessary complexity.